internal/helm/config.go

```diff
@@ -24,6 +24,7 @@ type Config struct {
 	DroneEvent         string   `envconfig:"DRONE_BUILD_EVENT"`      // Drone event that invoked this plugin.
 	UpdateDependencies bool     `split_words:"true"`                 // Call `helm dependency update` before the main command
 	AddRepos           []string `split_words:"true"`                 // Call `helm repo add` before the main command
+	RepoCAFile         string   `envconfig:"repo_ca_file"`           // CA certificate for `helm repo add`
```

Comment:
should we have this REPO_CA_FILE - just to keep it consistent with other all cap naming ? :)

Reply:
Most of the other settings are lowercase now, actually—I switched them over in #71 because I didn't like having the code yell at me. I agree they should consistent, though. Do you have a preference for upper/lower case?

Reply:
ah. honestly as long as it work and is the same. I'll go with whatever :)